src/System.Private.Uri/tests/FunctionalTests/IdnDnsSafeHostTest.cs

```diff
@@ -45,6 +45,16 @@ public void IdnDnsSafeHost_IPv6Host_ScopeIdButNoBrackets()
             Assert.Equal("[::1]", test.Host);
         }
 
+        [Fact]        
+        public void IdnDnsSafeHost_IPv6HostLinkLocalAddress_ScopeIdButNoBrackets()
+        {
+            Uri test = new Uri("http://[fe80::e077:c9a3:eeba:b8e9%18]/");
```

Comment:
You should refactor this duplicated string, "fe80::e077:c9a3:eeba:b8e9%18" into a constant in this test.

Reply:
i copy the pattern from test above https://github.com/MarcoRossignoli/corefx/blob/2d9eddafdf568afab7a8bb730415bfce9b885a15/src/System.Private.Uri/tests/FunctionalTests/IdnDnsSafeHostTest.cs#L41 do you confirm this refactoring?

Reply:
>i copied the pattern from test above https://github.com/MarcoRossignoli/corefx/blob/2d9eddafdf568afab7a8bb730415bfce9b885a15/src/System.Private.Uri/tests/FunctionalTests/IdnDnsSafeHostTest.cs#L41 do you confirm this refactoring?

Following existing code and test patterns can be useful.  But it doesn't work in all cases.  Sometimes there are bad patterns in the code. And we don't change them all, but try to add better code for new PRs.  Also, the pattern you cited uses simpler IP literals.  This test uses a long IPv6 literal. So, for code cleanliness and ease of maintenance, it's better to consolidate if possible.

Reply:
done with `[Theory]` could be useful for future new data.